apps/rocketchat/rocketchat-route-backup.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: route.openshift.io/v1
```

Comment:
remove the route files?

Reply:
whoops, there aren't supposed to be rocketchat files in here, let me fix that...

Reply:
okay those were just backups of the routes from previous outages of rc, my bad :P